.circleci/config.yml

```diff
@@ -346,6 +346,60 @@ jobs:
     - run: node ./scripts/circleci/setup-angular-snapshots.js
     - run: ./scripts/circleci/run-local-browser-tests.sh
 
+
+  # ----------------------------------------------------------------------------
+  # Job that runs all Bazel tests against Ivy with the current Angular version
+  # specified in the project dev dependencies.
+  # ----------------------------------------------------------------------------
+  ivy_test:
+    <<: *job_defaults
+    resource_class: xlarge
+    environment:
+      GCP_DECRYPT_TOKEN: *gcp_decrypt_token
+    steps:
+      - *checkout_code
+      - *restore_cache
+      - *copy_bazel_config
+      - *setup_bazel_remote_execution
+      - *yarn_install
+
+      # Setup Angular ivy snapshots built with ngtsc but locked to a specific tag. We
+      # cannot determine the tag automatically based on the Angular version specified in
+      # the "package.json" because the SHA is not known for the given release. Nor can
+      # we use ngcc to apply the ivy switches because ngcc currently does not handle the
+      # UMD format which is used by Bazel when running tests.
+      - run: node ./scripts/circleci/setup-angular-snapshots.js --tag 8.0.0-rc.0-ivy-aot+bd37622
```

Comment:
Unfortunately there is no other easy way to get a working Ivy NPM package output.. so for now we need to manually lock the tag here again.. ideally it would have been based on the Angular version specified in the `package.json`.

We can remove this logic once UMD support for NGC landed. This should be pretty soon.. so not worth putting to much effort into this IMO.

Reply:
@kara could we add a tag to angular/angular like `latest-release` / `most-recent-release`? It would be automatically applied by the release script. 

Otherwise I would add a link to a tracking issue for the umd thing

Reply:
@jelbourn This won't help either I think. What we really need is that the `publish-build-artifacts` script for framework pushes to the Github build repositories with a tag that equals the version name. 

Right now the snapshot tags always include a SHA and it's difficult to determine the proper tag. I even uses the Github API to find that SHA.. but it's not guaranteed to work **always** since releases can be made from patch branches where the Ivy output is not built.

https://github.com/angular/angular/blob/master/scripts/ci/publish-build-artifacts.sh#L127-L129

Reply:
The UMD PR is about to go in:

https://github.com/angular/angular/pull/25445

Alex says this will go in today, so seems like we can wait?

Reply:
In that case we should just wait. In the meanwhile we could still get this PR in and I'll create a follow-up tomorrow. I've addressed all feedback from @jelbourn and added a link for the UMD thing.

Reply:
sgtm